src/libs/auth.js

```diff
@@ -81,14 +104,13 @@ export const bucketBrowserUrl = id => {
 }
 
 export const initializeAuth = _.memoize(async () => {
-  await new Promise(resolve => window.gapi.load('auth2', resolve))
-  await window.gapi.auth2.init({ clientId: getConfig().googleClientId })
   const processUser = user => {
     return authStore.update(state => {
-      const authResponse = user.getAuthResponse(true)
-      const profile = user.getBasicProfile()
-      const isSignedIn = user.isSignedIn()
-      //The following few lines of code are to handle sign-in failures due to privacy tools.
+      const isSignedIn = user !== null && user !== undefined
```

Comment:
I might:
```suggestion
      const isSignedIn = !_.isNil(user)
```
just to aid readability here.